src/resume.json

```diff
@@ -39,30 +39,73 @@
   ],
 
   "skills": [
-    {
-      "name": "Web Development",
-      "keywords": [
-        "HTML",
-        "CSS",
-        "CSS Animations and Transitions",
-        "Responsive Design",
-        "Javascript",
-        "ESlint and Prettier",
-        "GitFlow",
-        "SEO",
-        "React"
-      ]
-    }
+    [
+      {
+        "name": "HTML",
+        "level": "Intermediate",
+        "logo": "html.png"
+      }
+    ],
+    [
+      {
+        "name": "CSS",
+        "level": "Intermediate",
+        "logo": "css.png"
+      }
+    ],
+    [{ "name": "Javascript", "level": "Intermediate", "logo": "js.png" }],
+    [{ "name": "React", "level": "Intermediate", "logo": "react.png" }],
+    [
+      {
+        "name": "CSS Animations and Transitions",
+        "level": "Intermediate",
+        "logo": "animation.png"
+      }
+    ],
+    [
+      {
+        "name": "ESlint and Prettier",
+        "level": "Intermediate",
+        "logo": "eslint.png"
+      }
+    ],
+    [
+      {
+        "name": "Gitflow",
+        "level": "Intermediate",
+        "logo": "git.png"
+      }
+    ],
+    [
+      {
+        "name": "SEO",
+        "level": "Begginer",
```

Comment:
typo: `Begginer` -> `Beginner` 
You can download this vs-code spell-checker plugin, to avoid future typos
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker